docs/postmortems/0000-00-00-template.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2024-01-01
+authors: spirit-developer
+---
+
+# Title of postmortem
+
+**Summary**
+
+<!-- What is a one or two-line summary of the event that occurred? -->
+
+**Impact**
+
+<!-- What was the scope of impact from the event occuring? How many teams were
+impacted? -->
+
+**Root causes**:
+
+<!-- Looking back, what ended up being the main reasons why this event occurred?
+-->
+
+**Detection**
+
+<!-- How did we find out or discover that this event had occurred? -->
+
+**Resolution**
+
+<!-- How did we end up addressing this event in order to mitigate impact? -->
+
+**Action Items**
+
+<!-- What are the action items that came out of this postmortem? Reference
+issues and Pull Requests in the "Bug" column with the appropriate owners -->
+
+| Action item | Owner | Bug |
+| ----------- | ----- | --- |
+
+## Lessons learned
+
+**What went well**
+
+- Item A
+- Item B
+- Item C
+
+**What went wrong**
+
+- Item A
+- Item B
+- Item C
+
+**Where we got lucky**
+
+- Item A
+- Item B
+- Item C
+
+## Timeline
+
+2024-01-01 **(all times in UTC)**
```

Comment:
do we actually need to write them in UTC? :D

Reply:
CET/CEST - https://time.is/CEST - This will be fun :-)